### PR TITLE
Db Filled with Data

### PR DESCRIPTION
**Pull request by Atalay-Polat**
Added data

### CODE REVIEW

File: exalearn-api/ExaLearn.WebApi/appsettings.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "AllowedHosts": "*",
```
Comment (romanstat) [inline exalearn-api/ExaLearn.WebApi/appsettings.json]: exclude file from git changes

File: exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs [hunk 2]
```diff
@@ -8,30 +8,116 @@ public static class ModelBuilderExtensions
     {
         public static void Seed(this ModelBuilder builder)
         {
+            FillBeginnerLevelWithData(builder);
+        }
+
+        public static void FillBeginnerLevelWithData(this ModelBuilder builder)
+        {
+            #region BeginnerGrammarQuestions
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs:16]: good)

**Review by romanstat — APPROVED** [review]

File: exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs [hunk 3]
```diff
@@ -8,30 +8,116 @@ public static class ModelBuilderExtensions
     {
         public static void Seed(this ModelBuilder builder)
         {
+            FillBeginnerLevelWithData(builder);
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs]: builder.FillBeginnerLevelWithData();?

Comment (Atalay-Polat) [reply exalearn-api/ExaLearn.Dal/Database/ModelBuilderExtensions.cs]: fixed

File: exalearn-api/ExaLearn.Dal/Migrations/20210803170603_db_filled_with_beginner_qa.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ExaLearn.Dal.Migrations
+{
+    public partial class db_filled_with_beginner_qa : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/Migrations/20210803170603_db_filled_with_beginner_qa.cs:8]: wht empty, delete migration

**Review by romanstat — APPROVED** [review]

**Review by vlad12345123 — APPROVED** [review]

**Pull request merged**